osu.Game/Screens/Play/HUD/ArgonHealthDisplay.cs

```diff
@@ -100,36 +101,45 @@ private void load()
             RelativeSizeAxes = Axes.X;
             AutoSizeAxes = Axes.Y;
 
-            InternalChild = new Container
+            InternalChildren = new[]
             {
-                AutoSizeAxes = Axes.Both,
-                Children = new Drawable[]
+                new Circle
                 {
-                    background = new BackgroundPath
-                    {
-                        PathRadius = main_path_radius,
-                    },
-                    glowBar = new BarPath
-                    {
-                        BarColour = Color4.White,
-                        GlowColour = main_bar_glow_colour,
-                        Blending = BlendingParameters.Additive,
-                        Colour = ColourInfo.GradientHorizontal(Color4.White.Opacity(0.8f), Color4.White),
-                        PathRadius = 40f,
-                        // Kinda hacky, but results in correct positioning with increased path radius.
-                        Margin = new MarginPadding(-30f),
-                        GlowPortion = 0.9f,
-                    },
-                    mainBar = new BarPath
+                    Position = new Vector2(-4f, main_path_radius - 1.5f),
+                    Size = new Vector2(50f, 3f),
+                },
+                new Container
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Margin = new MarginPadding { Left = 50f },
+                    Children = new Drawable[]
                     {
-                        AutoSizeAxes = Axes.None,
-                        RelativeSizeAxes = Axes.Both,
-                        Blending = BlendingParameters.Additive,
-                        BarColour = main_bar_colour,
-                        GlowColour = main_bar_glow_colour,
-                        PathRadius = main_path_radius,
-                        GlowPortion = 0.6f,
-                    },
+                        background = new BackgroundPath
+                        {
+                            PathRadius = main_path_radius,
+                        },
+                        glowBar = new BarPath
+                        {
+                            BarColour = Color4.White,
+                            GlowColour = main_bar_glow_colour,
+                            Blending = BlendingParameters.Additive,
+                            Colour = ColourInfo.GradientHorizontal(Color4.White.Opacity(0.8f), Color4.White),
+                            PathRadius = 40f,
+                            // Kinda hacky, but results in correct positioning with increased path radius.
+                            Margin = new MarginPadding(-30f),
+                            GlowPortion = 0.9f,
+                        },
```

Comment:
Please implement the line outside of this class and revert these changes.

Reply:
I was going to bring this back in a separate class, but after applying https://github.com/ppy/osu/pull/25226#discussion_r1375789990 as well, I think it's better to just remove it altogether and have the health bar fill up the entire screen...

![CleanShot 2023-11-05 at 01 34 10](https://github.com/ppy/osu/assets/22781491/0cc90e61-6d69-496a-9ba3-df687eb0d08e)

Reply:
Line removed in https://github.com/ppy/osu/pull/25226/commits/1c844a155e97059e23cdae6470401d217fa88db3